src/darwin/Framework/CHIP/MTRDeviceController.h

```diff
@@ -178,16 +178,7 @@ MTR_AVAILABLE(ios(16.1), macos(13.0), watchos(9.1), tvos(16.1))
                                                         error:(NSError * __autoreleasing *)error
     MTR_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
 
-/**
- * Optionally pre-warm the controller for setting up a commissioning session.
- * This may be called before setupCommissioningSessionWithPayload if it's known
- * that a commissioning attempt will soon take place but the commissioning
- * payload is not known yet.
- *
- * For example this may do a BLE scan in advance so results are ready earlier
- * once the discriminator is known.
- */
-- (void)preWarmCommissioningSession MTR_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
+- (void)preWarmCommissioningSession MTR_NEWLY_DEPRECATED("-[MTRDeviceControllerFactory preWarmCommissioningSession]"); // MTR_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
```

Comment:
```suggestion
- (void)preWarmCommissioningSession MTR_NEWLY_DEPRECATED("-[MTRDeviceControllerFactory preWarmCommissioningSession]") MTR_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
```
so we are ok even if we somehow ship this without fixing the `MTR_NEWLY_DEPRECATED` bit.  You can have both annotations present, especiall since MTR_NEWLY_DEPRECATED expands to nothing, right?

Reply:
Ah yes, it's just `_MTR_IMPLICIT_EXPORT` so it's fine to have both.